tests/ikev2-weak-dh/test.yaml

```diff
@@ -31,3 +32,39 @@ checks:
         ikev2.payload[1]: KeyExchange
         ikev2.payload[2]: SecurityAssociation
         ikev2.payload[3]: NoNextPayload
+
+  - filter:
+      count: 1
+      version: 5
```

Comment:
Can you not do a max-version 5 ?

Reply:
don't think so, in the run.py is only ```version``` and ```min-version``` defined